library/src/main/java/com/getstream/sdk/chat/viewmodel/MessageInputViewModel.kt

```diff
@@ -102,6 +102,10 @@ class MessageInputViewModel(private val cid: String, private val chatDomain: Cha
 		chatDomain.useCases.sendMessage.invoke(message).execute()
 	}
 
+	fun sendMessage(message: String) {
+		chatDomain.useCases.sendMessage(Message(cid = cid, text = message)).execute()
```

Comment:
`chatDomain` could be just called `domain`. It's the only entity with such name in the project and chat prefix is excessive.